### ISSUE TITLE
Expose ncurses setlocale or call it on initialization

### ISSUE THREAD

**Issue by gyscos** [ihalila/pancurses#8]
In order to properly support unicode output on linux, `ncurses::setlocale` has to be called at the beginning of the program (before `initscr()`).
This is not currently exposed by pancurses.
Possible solutions:
* Expose a `setlocale()` method, similar to [ncurses-rs](https://github.com/jeaye/ncurses-rs/blob/master/src/ncurses.rs#L1193)
* Run the following code in `pancurses::initscr()`:

    ```rust
    let buf = CString::new("").unwrap().as_ptr();
    unsafe { curses::setlocale(0, buf) }; // The magic number 0 corresponds to LC_ALL
    ```
    * Always
    * Only when some feature is activated

**Comment by ihalila** [comment]
I added a platform specific pre_init() function in 16139e4fa7318e6fde742b7c93ba169adb424ee5 (don't mind the time travel, my virtual machine's date was wrong), setlocale() gets called for the ncurses backend now, it ought to fix things.

**Comment by gyscos** [comment]
:+1: 
Just waiting on the crates.io version :)

**Comment by ihalila** [comment]
I published 0.6, should be available from crates.io now.
